src/dearpygui.cpp

```diff
@@ -544,12 +554,15 @@ PyInit__dearpygui(void)
 	MV_ADD_COMMAND(clear_selected_links);
 
 	// plots
-	MV_ADD_COMMAND(is_plot_queried);
```

Comment:
`is_plot_queried`:  To make the change "less breaking", we can keep this function in `_deprecated.py` as a thin wrapper around `get_plot_query_rects`. It should return `True` if the list obtained from `get_plot_query_rects` is not empty. This won't be exactly identical to the way `plot.Queried` is set in the old version, but should be "good enough" for most applications to work until they get migrated to `get_plot_query_rects`.